### DISCUSSION TITLE
npx @modern-js/create@latest - Creating app without interactive inputs

### DISCUSSION THREAD

**Discussion by raolak** [web-infra-dev/modern.js#6581] [Q&A] [answered]
Hello

I want to automate app creation by passing arguments to npx @modern-js/create@latest so that I dont need to manually feed language and packageManager options... Also once this is done , I need do same for pnpm run new and enabling features such as bff, microfrontend etc...

is there any way to achieve this without user inputs and passing some sort of config ?

Isee this is from debug lof... but i dont know if similar config can be passed through cli

 Please select the programming language: TS
? Please select the package manager: pnpm
  create:debug 💡 [Input Answer]: {"locale":"en","defaultBranch":"main","packagesInfo":{},"isMwa":true,"isEmptySrc":true,"language":"ts","packageManager":"pnpm"} +10s
🚶  Load Generator...
  create:debug 💡 [Load Remote Generator]: @modern-js/base-generator +0ms
  create:debug 🌟 [Load Remote Generator Success]: @modern-js/base-generator 3.7.6 +0ms
  create:debug 🚀 [Start Run Base Generator] +110ms
  create:debug 💡 [Current Config]: {"locale":"en","defaultBranch":"main","packagesInfo":{},"isMwa":true,"isEmptySrc":true,"language":"ts","packageManager":"pnpm"} +0ms
  
  
  
  Expectation
  -  npx @modern-js/create@latest - should enable automation without taking user inputs
  - pnpm run new - should enable automation without takling user inputs

**Comment by caohuilin** [comment, marked as answer]
@raolak you can use --config params: `npx @modern-js/create@latest --config '{"language":"ts","packageManager":"pnpm"}'`
new command also support --config params: `pnpm run new --config '{"actionType":"function","function":"tailwindcss"}'`
